kernel_gateway/gatewayapp.py

```diff
@@ -152,9 +153,16 @@ def _default_kernel_name_default(self):
     def _list_kernels_default(self):
         return os.getenv(self.list_kernels_env, 'False') == 'True'
 
+    api_env = 'KG_API'
+    api = Unicode(config=True,
+        help='Designates that the kernel gateway will serve APIs from this notebook (KG_API env var)'
```

Comment:
The two options are separate. The seed notebook makes sense under both API modes (and potentially new modes that will be added.)